backpack/custom_module/permute.py

```diff
@@ -0,0 +1,29 @@
+"""Module containing Permute module."""
+from typing import Any
+
+from torch import Tensor
+from torch.nn import Module
+
+
+class Permute(Module):
+    """Module to permute a tensor."""
+
+    def __init__(self, dims: Any):
```

Comment:
Would it be better to share the interface with [`torch.Tensor.permute`](https://pytorch.org/docs/stable/tensors.html#torch.Tensor.permute)?

Reply:
Interface of ``permute`` is ``*dims: Any``. Changed it to that.